charts/castai-cluster-controller/values.yaml

```diff
@@ -54,10 +54,16 @@ additionalEnv:
 
 resources:
   requests:
-    cpu: 50m
-    memory: 100Mi
+    memory: 256Mi
+    cpu: 100m
   limits:
-    memory: 1Gi
+    memory: 256Mi
+    cpu: 100m
```

Comment:
If VPA is enabled, the resources should be null because they will be adjusted automatically during runtime. Otherwise tools that depend on the diff (like argocd) will always find a mismatch.

Reply:
Done as suggested